src/entries/demo/Demo.js

```diff
@@ -30,6 +30,7 @@ export class Demo extends React.PureComponent {
   render () {
     const { classes } = this.props
     const { definitionUrl } = this.state
+    const initialSchemaTreeDepth = 1
```

Comment:
Lol. I can't think of a better one. Potentially we will also have `initialJsonTreeDepth`.